From 858a89f9f4ebb65938c4f595640af78b202a776d Mon Sep 17 00:00:00 2001
From: Evan Layton <evan.layton@nexenta.com>
Date: Wed, 26 Sep 2018 13:23:12 -0600
Subject: [PATCH] NEX-18716 The NFS grace period is too long causing long
 failover times. Reviewed by: Joyce McIntosh <joyce.mcintosh@nexenta.com>
 Reviewed by: Cynthia Eastham <cynthia.eastham@nexenta.com> Reviewed by: Rick
 McNeal <rick.mcneal@nexenta.com>

---
 usr/src/cmd/fs.d/nfs/svc/nlockmgr.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/cmd/fs.d/nfs/svc/nlockmgr.xml b/usr/src/cmd/fs.d/nfs/svc/nlockmgr.xml
index b5165e7ad5..9ca641d319 100644
--- a/usr/src/cmd/fs.d/nfs/svc/nlockmgr.xml
+++ b/usr/src/cmd/fs.d/nfs/svc/nlockmgr.xml
@@ -93,7 +93,7 @@
 	</property_group>
 	<instance name='default' enabled='false'>
 	  <property_group name='nfs-props' type='com.oracle.nfs,props'>
-	    <propval name='grace_period' type='integer' value='90'/>
+	    <propval name='grace_period' type='integer' value='30'/>
 	    <propval name='lockd_listen_backlog' type='integer' value='32'/>
 	    <propval name='lockd_retransmit_timeout' type='integer' value='5'/>
 	    <propval name='lockd_servers' type='integer' value='256'/>
-- 
2.17.2 (Apple Git-113)

